service/src/main/kotlin/fi/espoo/evaka/application/ApplicationForm.kt

```diff
@@ -255,8 +255,8 @@ data class ApplicationForm(
                         postOffice = guardian.postOffice ?: ""
                     ),
                     futureAddress = null,
-                    phoneNumber = "",
-                    email = ""
```

Comment:
Noticed that I'm not currently storing whether the `noEmail` checkbox is checked. I'll fix that.

Reply:
On the other hand. This may be something we don't event want to store.

Reply:
I guess when we map backend data to form data we can interpret missing email so that the checkbox is on

Reply:
no need for a separate flag in backend/db imho

Reply:
Wouldn't that default to the checkbox being checked on creating a new application. We probably don't want that?

The current implementation would require an user that does not have an email address to re-check the checkbox on each update of the application. It sounds like an edge case to me: maybe we should just proceed with this one?

Reply:
Ah yeah, true that.. Ok then, I guess we can go with "it's not a bug it's a feature". :D